element_calcium_imaging/imaging_no_curation.py

```diff
@@ -971,15 +975,16 @@ class ActivityExtractionMethod(dj.Lookup):
 class Activity(dj.Computed):
     definition = """  # inferred neural activity from fluorescence trace - e.g. dff, spikes
     -> Fluorescence
-    -> ActivityExtractionMethod
+    -> ActivityExtractionParamSet
```

Comment:
I could be mistaken, but I don't think this table is defined.

Reply:
yes, a mistake, fixed